sycl/test-e2e/Graph/Explicit/enqueue_ordering.cpp

```diff
@@ -18,6 +18,10 @@ int main() {
     return 0;
   }
 
+  if (!Queue.get_device().has(sycl::aspect::usm_shared_allocations)) {
```

Comment:
Nit; The right way to make this requirement now would be to do
```c++
// REQUIRES: aspect-usm_shared_allocations
```

Reply:
This test has been updated this way.